property_tests/are_equal/index.ts

```diff
@@ -1,2 +1 @@
-export * from './float';
 export * from './principal';
```

Comment:
Why are we exporting all? I don't think we should export all in a developer-facing thing like this